README.md

```diff
@@ -26,6 +26,8 @@ A list of students from Mount Sion CBS who actively contribute to the project
 * **Maciej Marchel** [Github](https://github.com/maciejmarchel12)
 * **Dominik Kawka** [Github](https://github.com/dominikkawka)
 * **Kieran Hanrahan** [Github](https://github.com/kieranhanrahan)
+* **Evan Hearne** [Github](https://github.com/evanhearne)
```

Comment:
@evanhearne it would be nice to split this out to phases as this list will probably grow even further over time.

Reply:
@laurafitzgerald No, problem. Will do that now :)

Reply:
OK, I split the contributors between phase 1 and 2 :) @laurafitzgerald